cmds/tui.c

```diff
@@ -175,6 +179,15 @@ static void init_colors(void)
 	init_pair(C_GREEN,  COLOR_GREEN,  COLOR_BLACK);
 	init_pair(C_YELLOW, COLOR_YELLOW, COLOR_BLACK);
 	init_pair(C_RED,    COLOR_RED,    COLOR_BLACK);
+
+	if (init_color(COLOR_CYAN, 128, 128, 128) == ERR) {
```

Comment:
It seems that this `init_color` always failed in my environment for some reasons.

Reply:
Thank you for your opinion.
I tested it after changing environment variable (TERM=xterm-256color) as referring page below.
https://stackoverflow.com/questions/47686906/ncurses-init-color-has-no-effect
Thank you.